docs/getting-started.md

```diff
@@ -455,18 +443,20 @@ Click "Next" then "Finish" to create your AVD. At this point you should be able
 
 <h2>Running your React Native application</h2>
 
-Run `react-native run-ios` inside your React Native project folder:
+> If you use Yarn package manager, your can use `yarn` instead of `npx` when running React Native commands inside an existing project.
```

Comment:
```suggestion
> If you use the Yarn package manager, you can use `yarn` instead of `npx` when running React Native commands inside an existing project.
```